department-of-veterans-affairs/va.gov-cms
# VACMS-6935 - Create metatag audit view

## Description
Created a view to help audit the current meta tags and how they're being used across all Veteran-facing content types.

Relates to #6935.

## Testing done
Navigated the view and saw the various fields being displayed for meta tags within the view results list.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/146839536-3c1bd1fe-691c-465e-bd76-a5248f04159e.png)


## QA steps

As cms editor I can
1. View the metatag audit tool (/admin/content-tools/metatag-audit) and
1. View a list of content and the meta data stored in the CMS
1. Then validate Acceptance Criteria from issue
- [x] See what metadata drupal currently has (and what it has to offer that isn’t currently used)

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
